README.md

```diff
@@ -1,60 +1,137 @@
+Deploy anywhere Node.js runs, or choose [ApostropheCMS hosting](https://apostrophecms.com/hosting) for turnkey solutions with optimized performance and dedicated support.
 
-| Software | Minimum Version | Notes
-| ------------- | ------------- | -----
-| Node.js | 12.x | Or better
-| npm  | 6.x  | Or better
-| MongoDB  | 3.6  | Or better
-| Imagemagick  | Any | Faster image uploads, GIF support (optional)
+## Built With Modern Tech
 
-## Community
+- **[Node.js](https://nodejs.org/)** - JavaScript runtime for server-side development
+- **[MongoDB](https://www.mongodb.com/)** - Flexible document database for content storage
+- **ESM Modules** - Native ES6 module support for modern JavaScript
+- **Vite** - Lightning-fast build tool and development server
+- **Modern JavaScript** - ES6+, async/await, and contemporary development patterns
 
-[Discord](https://discord.com/invite/XkbRNq7) - [Twitter](https://twitter.com/apostrophecms) - [Discussions](https://github.com/apostrophecms/apostrophe/discussions)
+## Community & Support
+
+**Join other developers and content creators using ApostropheCMS:**
+
+- **[Discord](https://discord.com/invite/XkbRNq7)** - Get help, share projects, and connect with other users
+- **[GitHub Discussions](https://github.com/apostrophecms/apostrophe/discussions)** - Feature requests, technical discussions, and community support
+- **[Documentation](https://docs.apostrophecms.org/)** - Comprehensive guides, tutorials, and API references
 
 ## Contributing
 
-We eagerly welcome open source contributions. Before submitting a PR, please read through our [Contribution Guide](https://github.com/apostrophecms/apostrophe/blob/main/CONTRIBUTING.md)
+We welcome contributions from the community! Whether you're fixing bugs, adding features, or improving documentation, your help makes ApostropheCMS better for everyone.
+
+- **[Contribution Guide](https://github.com/apostrophecms/apostrophe/blob/main/CONTRIBUTING.md)** - How to contribute code, documentation, and feedback
+- **[Good First Issues](https://github.com/apostrophecms/apostrophe/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)** - Perfect starting points for new contributors
+- **[Development Setup](https://docs.apostrophecms.org/guide/development-setup.html)** - Set up your local development environment
+
+
+## Pro Features
+
+**For teams and organizations requiring additional features:**
+
+- **🔐 Advanced User Management** - Granular permissions, user groups, and access controls
+- **🌍 Automated Translation** - AI-powered translation with DeepL, Google Translate, and Azure
+- **📊 Analytics & SEO** - Built-in SEO optimization and content analytics
+- **⚡ Performance Optimization** - Advanced caching, CDN integration, and performance monitoring
+- **💼 Professional Support** - Dedicated support, training, and consultation services
+
+[Contact us](https://apostrophecms.com/contact-us) to learn about pro licensing and support options.
```

Comment:
I would link them to our extensions page. I'd also mention multisite capabilities.

We have a /pro landing page but I think it needs love.